site/app/templates/admin/users/MoreExtensions.twig

```diff
@@ -17,5 +17,5 @@
 {% block close_button_text %}No{% endblock %}
 {% block buttons %}
     {{ block('close_button') }}
-    <a class="btn btn-primary" id="apply-to-all" href="javascript:confirmExtension(1)">Yes</a>
+    <a class="btn btn-primary" id="apply-to-all" data-testid='more-extension-apply-to-all' href="javascript:confirmExtension(1)">Yes</a>
```

Comment:
```suggestion
    <a class="btn btn-primary" id="apply-to-all" data-testid="more-extension-apply-to-all" href="javascript:confirmExtension(1)">Yes</a>
```

I don't think that his matters in a syntax point of view, just a consistency point of view

Reply:
I noticed that there were a couple of places where we used example_test_id and some places where we used example-test-id

out of consistency which ones better?

Reply:
Originally we just did whatever, but I am trying to encourage using example-test-id to be consistent.